index.bs

```diff
@@ -93,6 +96,8 @@ dictionary MediaStreamTrackProcessorInit {
 2. Let |p| be a new {{MediaStreamTrackProcessor}} object.
```

Comment:
Might be wise to call this something other than |p|, since |p| is frequently used to refer to a promise that is going to be returned from the algorithm.

Reply:
Done